README.md

```diff
@@ -130,6 +130,8 @@ var WebSocketServer = require('ws').Server
 
 wss.broadcast = function broadcast(data) {
   wss.clients.forEach(function each(client) {
+    // use this, if you want to send to everyone else
+    // if (client !== ws) client.send(data);
```

Comment:
The only thing I'm not sure is that the reader may wonder what `ws` is as it isn't defined in this example, otherwise looks good.